Documentation/Dev/Database Design.md

```diff
@@ -0,0 +1,123 @@
+# Database Design
+This is a summary of the different entities and relationships extracted from our functional requirements.
+I'd like to add the MySQL Datatypes to this but I'm not sure if that's overkill. 
+## Entities
```

Comment:
This is a fairly extensive list, nice work. We'll stick to the requirements for this milestone when actually creating the database for now, though.